src/main/webapp/js/client.js

```diff
@@ -0,0 +1,157 @@
+// const loadingElement = document.querySelector(".loading");
+const dataElement = document.querySelector(".data");
+
+// loadingElement.style.display = "none";
+
+function listAllFeeds() {
+    getFeeds().then((feeds) => {
+        dataElement.innerHTML = "";
+        feeds.forEach((feed) => {
+            const div = document.createElement("div");
+
+            const header = document.createElement("h3");
+            header.textContent = `${feed.restaurant.name} & ${feed.charity.name}`;
+
+            const contents = document.createElement("p");
```

Comment:
SHould we call it description

Reply:
Yes, this JS code segment was written by @anh2111htd. @KhawYewOnn and I already suggested this change, which should now be reflected in consequent PRs.